### ISSUE TITLE
Apply to join GNOME Circle

### ISSUE THREAD

**Issue by julianfairfax** [d-k-bo/televido#5]
Applying to join [GNOME Circle](https://apps.gnome.org/en-GB/#circle) would give the app greater visibility within the GNOME ecosystem.

**Comment by d-k-bo** [comment]
Thank you for your suggestion. At this point, I think the project is still a bit too immature and needs some more polish. I'll keep it in mind and will re-evaluate later.

**Comment by julianfairfax** [comment]
In addition to joining Circle, it would also be nice if the app's code were moved to [GNOME World](https://gitlab.gnome.org/World). GitHub is not an open source platform, and being owned by Microsoft makes it more of a danger to open source projects, rather than a help.

**Comment by d-k-bo** [comment]
I am aware of the problems with GitHub, but I think it is still the best place for this project. The [MediathekView](https://github.com/mediathekview/) project lives here, the Rust ecosystem is unfortunately fully dependent on GitHub and other platforms are too fragmented IMO.

I'm thinking about moving my projects to a different platform once GitLab and/or Forgejo/Gitea support federated issues and PRs via [ForgeFed](https://forgefed.org/).

**Comment by Vistaus** [comment]
GitLab is always a bit finicky for me, but I wouldn't oppose moving there. Although I do prefer GitHub. But GNOME World would mean that the translations would be subject to Damned Lies. I know I'm not the only person in the FOSS community that has issues with the team members of the respective language, not to mention it's not a user-friendly translation platform. And they already have a lot to translate with not enough members in some languages, which doesn't do the quality justice, but it also means some non-core projects get left behind.

tl;dr I prefer GitHub, but if you really want to move to GitLab, I'm not opposing that. I would oppose moving to GNOME World and Damned Lies, however.

**Comment by d-k-bo** [comment]
> Damned Lies members are against people manually updating the po files directly on GitLab

That is a no-go for me too.
